src/Avalonia.Controls/Navigation/IBiDirectionalNavigationRouter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Avalonia.Controls
+{
+    public interface IBiDirectionalNavigationRouter : INavigationRouter
```

Comment:
Why this interface?

Reply:
To provide support for navigating forward in the control. The default router doesn't support moving forward as it assume typical mobile navigation, which don't usually allow moving forward. This interface is provided so users can implement forward navigation logic, and the control will adapt to it.

Reply:
I would move it to INavigationRouter to avoid double checking of interface on NavigationControl